Automattic/newspack-theme
# fix(ras-acc): merge trunk into epic/ras-acc

### All Submissions:


### Changes proposed in this Pull Request:

This PR is to merge trunk into the epic/ras-acc branch after the dependency updates. 

### How to test the changes in this Pull Request:

1. Run `npm run build` and confirm there are no errors.
2. Spot check the updates and confirm that nothing looks out of place.

### Other information:
